androidApp/build.gradle.kts

```diff
@@ -29,18 +29,17 @@ android {
     }
 
     buildTypes {
-        create("debugPG") {
```

Comment:
why did you remove the custom build type? it was for a test proguard build

Reply:
Ah, is that what it was!